docs/api/fleet.asciidoc

```diff
@@ -0,0 +1,11 @@
+[role="xpack"]
+[[fleet-api]]
+== Kibana Fleet APIs
+
+Manage your Fleet Agents.
```

Comment:
```suggestion
Manage your Fleet agents.
```

Reply:
I'm unsure about our rules for capitalization here, but I prefer lowercase "agent". I see both in this document, so I've edited to reflect that. The style guide is a little murky for our usage:

> Agent, agent
> Agent is a proper noun when it is part of the name of an Elastic product name. These agent names should always be preceded by Elastic.
> 
> Elastic APM RUM Agent
> Elastic APM Node.js Agent
> 
> When used in a general sense, agent is not capitalized.
> 
> APM agents are open source libraries written in the same language as your service.

Reply:
@gchaps WDYT? If Fleet Agent is the new name for Beats Agent, then maybe it should be capitalized, but I'm unsure because I'm a bit out of the loop.

Reply:
@chandlergibb What is the proper way to refer to Fleet Agent?